open3d_slam/include/open3d_slam/cloud_processing.hpp

```diff
@@ -0,0 +1,20 @@
+/*
+ * CloudProcessing.hpp
+ *
+ *  Created on: Oct 21, 2022
+ *      Author: jelavice
+ */
+
+#pragma once
+#include <open3d/geometry/PointCloud.h>
+#include "open3d_slam/typedefs.hpp"
+#include <memory>
+
+namespace o3d_slam {
+
+class CroppingVolume;
```

Comment:
Not to be addressed in this PR but I would support for a name like VolumeCropper, but also doesn't affect much

Reply:
Yes, that is actually a better name.